tests/surrogate_test.py

```diff
@@ -1,426 +1,1457 @@
 import numpy
 import pytest
 
-from smolyay.basis import ChebyshevFirstKind
-from smolyay.grid import SmolyakGridGenerator, IndexGrid
-from smolyay.surrogate import Surrogate, GradientSurrogate
+import smolyay
+
+
+def create_surrogate(
+    surrogate_class,
+    nested_basis_set_class,
+    num_level,
+    domain,
+    nested_point_class=None,
```

Comment:
This can't be none, or it was choke right?
```suggestion
    nested_point_class,
```